src/main/java/com/zestedesavoir/zestwriter/view/dialogs/OptionsDialog.java

```diff
@@ -0,0 +1,229 @@
+package com.zestedesavoir.zestwriter.view.dialogs;
+
+
+import com.zestedesavoir.zestwriter.MainApp;
+import com.zestedesavoir.zestwriter.utils.Configuration;
+import javafx.fxml.FXML;
+import javafx.scene.control.*;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.stage.Stage;
+
+import java.util.Optional;
+
+public class OptionsDialog{
+    private MainApp mainApp;
+    private Stage optionsWindow;
+    private Configuration config;
+
+    @FXML private Hyperlink optionGeneral;
+    @FXML private Hyperlink optionEditor;
+    @FXML private Hyperlink optionDisplay;
+    @FXML private Hyperlink optionShortcut;
+    @FXML private Hyperlink optionAuthentification;
+    @FXML private Hyperlink optionAdvanced;
+
+    @FXML private AnchorPane optionGeneralPane;
+    @FXML private AnchorPane optionEditorPane;
+    @FXML private AnchorPane optionDisplayPane;
+    @FXML private AnchorPane optionShortcutPane;
+    @FXML private AnchorPane optionAuthentificationPane;
+    @FXML private AnchorPane optionAdvancedPane;
+
+    @FXML private ComboBox<String> optEditorFont;
+    @FXML private ComboBox<Integer> optEditorFontSize;
+    @FXML private ComboBox<String> optDisplayTheme;
+    @FXML private TextField optAuthentificationUsername;
+    @FXML private TextField optAuthentificationPassword;
+    @FXML private ComboBox<String> optAdvancedProtocol;
+    @FXML private TextField optAdvancedHost;
+    @FXML private TextField optAdvancedPort;
+
+    public enum EditorFonts{
+        Arial("Arial"),
```

Comment:
Oui on peut utiliser ControlFX avec gradle. Il te suffit de rajouter le nom
de la lib et sa version dans le build.gradle.

Le dim. 17 avr. 2016 01:19, WinXaito <notifications@github.com> a écrit :

> In
> src/main/java/com/zestedesavoir/zestwriter/view/dialogs/OptionsDialog.java
> <https://github.com/firm1/zest-writer/pull/53#discussion_r59975629>:
>
> > +    @FXML private AnchorPane optionDisplayPane;
> > +    @FXML private AnchorPane optionShortcutPane;
> > +    @FXML private AnchorPane optionAuthentificationPane;
> > +    @FXML private AnchorPane optionAdvancedPane;
> > +
> > +    @FXML private ComboBox<String> optEditorFont;
> > +    @FXML private ComboBox<Integer> optEditorFontSize;
> > +    @FXML private ComboBox<String> optDisplayTheme;
> > +    @FXML private TextField optAuthentificationUsername;
> > +    @FXML private TextField optAuthentificationPassword;
> > +    @FXML private ComboBox<String> optAdvancedProtocol;
> > +    @FXML private TextField optAdvancedHost;
> > +    @FXML private TextField optAdvancedPort;
> > +
> > +    public enum EditorFonts{
> > +        Arial("Arial"),
>
> J'ai pris contact avec le rédacteur du tutoriel. Pour utiliser le Font
> Selector, il faut soit utiliser la librairie ControlFX, soit créer la
> fenêtre de sélection nous même, car il n'est pas intégré de base dans
> JavaFX.
>
> Y a t'il moyen d'utiliser ControlFX avec Gradle ?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/firm1/zest-writer/pull/53/files/4fb56f6425a5feee48289f04e1acfee821c1b9cb#r59975629>
>